src/translations.php

```diff
@@ -201,6 +201,15 @@
         "Longest Streak" => "En Uzun Seri",
         "Present" => "Şu an",
     ],
+    "ua" => "uk_UA",
+    "uk_UA" => [
```

Comment:
It seems like `uk` would be the correct [locale code](https://gist.github.com/DenverCoder1/f61147ba26bfcf7c3bf605af7d3382d5) for Ukrainian.

ua does not seem to appear on the list and since there's only one major version of Ukrainian, the country code is not needed.

```suggestion
    "uk" => [
```

If you think this sounds right, feel free to use the "commit suggestion" button or push to your branch to update this.